docs/guide/browser/assertion-api.md

```diff
@@ -32,35 +32,17 @@ Vitest bundles the [`@testing-library/jest-dom`](https://github.com/testing-libr
 - [`toHaveRole`](https://github.com/testing-library/jest-dom#toHaveRole)
 - [`toHaveErrorMessage`](https://github.com/testing-library/jest-dom#toHaveErrorMessage)
 
-If you are using TypeScript or want to have correct type hints in `expect`, make sure you have either `@vitest/browser/providers/playwright` or `@vitest/browser/providers/webdriverio` specified in your `tsconfig` depending on the provider you use. If you use the default `preview` provider, you can specify `@vitest/browser/matchers` instead.
+If you are using [TypeScript](/guide/browser/#typescript) or want to have correct type hints in `expect`, make sure you have either `@vitest/browser/providers/playwright` or `@vitest/browser/providers/webdriverio` referenced in your tests depending on the provider you use. If you use the default `preview` provider, you can specify `@vitest/browser/matchers` instead.
 
 ::: code-group
-```json [preview]
-{
-  "compilerOptions": {
-    "types": [
-      "@vitest/browser/matchers"
-    ]
-  }
-}
+```ts [preview]
+/// <reference types="@vitest/browser/matchers" />
 ```
-```json [playwright]
-{
-  "compilerOptions": {
-    "types": [
-      "@vitest/browser/providers/playwright"
-    ]
-  }
-}
+```ts [playwright]
+/// <reference types="@vitest/browser/providers/playwright" />
```

Comment:
Alas, this doesn't give type definitions for asymmetric matchers, like `expect.toHaveTextContent()`, which are still there on runtime. Are we missing it in the `.d.ts` itself, or an extra step is needed for it to work?

Reply:
My bad! Saw the fix in https://github.com/vitest-dev/vitest/pull/6934/files#diff-826b776cbcfb2b064b48a22493ccfbee249a739d2ac0332267f44f30a767e49cR7. 👏